internal/cache/clockpro.go

```diff
@@ -329,13 +329,13 @@ func (c *shard) metaDel(e *entry) {
 	}
 
 	if e == c.handHot {
-		c.handHot = c.handHot.prev()
+		c.handHot = c.handHot.next()
```

Comment:
I think when entry is removed, hands should move to next. If hands move backward, it means testing same entry two times in a row. For example, handHot resets the reference bit of entry A and moves to entry B. However the entry B is removed soon and handHot moves back and point the entry A again. It leads the entry A can be demoted to a cold list very fast, even it has a very small reuse distance.